pulp_docker/app/serializers.py

```diff
@@ -13,6 +13,111 @@
 from . import models
 
 
+class ManifestListTagSerializer(platform.ContentSerializer):
+    """
+    Serializer for ManifestListTags.
+    """
+
+    name = serializers.CharField(help_text="Tag name")
+    manifest_list = platform.DetailRelatedField(
+        many=False,
+        help_text="Manifest List that is tagged",
+        view_name='docker-manifest-lists-detail',
+        queryset=models.ImageManifest.objects.all()
+    )
+
+    class Meta:
+        fields = tuple(
+            set(platform.ContentSerializer.Meta.fields) - {'artifacts'}
+        ) + ('name', 'manifest_list')
+        model = models.ManifestListTag
+
+
+class ManifestTagSerializer(platform.ContentSerializer):
+    """
+    Serializer for ManifestTags.
+    """
+
+    name = serializers.CharField(help_text="Tag name")
+    manifest = platform.DetailRelatedField(
+        many=False,
+        help_text="Manifest that is tagged",
+        view_name='docker-manifests-detail',
+        queryset=models.ImageManifest.objects.all()
+    )
+
+    class Meta:
+        fields = tuple(
+            set(platform.ContentSerializer.Meta.fields) - {'artifacts'}
```

Comment:
A recent change https://pulp.plan.io/issues/4206 means that this should be `_artifacts`